build/js/cookie-policy.js

```diff
@@ -9,74 +7,111 @@ if (typeof ubuntu === 'undefined') {
 
 if (ubuntu.hasOwnProperty('cookiePolicy')) {
   throw TypeError("Namespace 'ubuntu' not available");
-}
+} // The cookie policy injection and interaction
 
-// The cookie policy injection and interaction
-ubuntu.cookiePolicy = function () {
+ubuntu.cookiePolicy = (function() {
```

Comment:
I think we should use a generic namespace instead of using as this plugin may be used on a none Ubuntu site.